scenario/consistency_checker.py

```diff
@@ -529,8 +530,13 @@ def check_containers_consistency(
         if evt_container_name not in state_containers:
             errors.append(
                 f"the event being processed concerns container {evt_container_name!r}, but a "
-                f"container with that name is not present in the state. It's odd, but consistent, "
-                f"if it cannot connect; but it should at least be there.",
+                f"container with that name is not present in the state. It's odd, but "
+                f"consistent, if it cannot connect; but it should at least be there.",
```

Comment:
FYI `tox -e fmt` must have done this wrapping - I didn't do it deliberately. I can revert it if you prefer.

Reply:
I don't mind, but that makes me realize, I think the f-string without placeholders we have at lines 532/533 should be linter errors. Can you dig into why they are not? 
Could also refactor this as a triple-quoted multiline fstring if that helps break it down more naturally.

Reply:
> think the f-string without placeholders we have at lines 532/533 should be linter errors. Can you dig into why they are not?

F541 is [deliberately only applied to the final string when using implicit string concatenation](https://github.com/astral-sh/ruff/issues/4662). Mostly to make the alignment match, I think (based on that ticket and another ruff one).

Reply:
huh! makes sense. Thanks for checking :)